installers-conda/build-environment.yml

```
@@ -2,13 +2,13 @@ name: spy-inst
 channels:
   - conda-forge
 dependencies:
-  - conda >=24.5.0
-  - conda-build >=24.5.0
-  - conda-lock >=2.5.7
-  - conda-standalone >=24.5.0
-  - constructor >=3.6.0
+  - conda =24.5.0
+  - conda-build =24.5.1
+  - conda-lock =2.5.7
+  - conda-standalone =24.5.0
+  - constructor =3.8.1
   - gitpython
   - mamba
-  - menuinst >=2.1.0
+  - menuinst =2.1.1
```

There are not significant changes from 2.1.0 to 2.1.1, so we can pin 2.1.0 for rc1 and then pin to 2.1.1 for rc2 and check that the base environment was updated.

```suggestion
  - menuinst =2.1.0
```